webrender_api/src/font.rs

```diff
@@ -274,3 +267,53 @@ pub struct GlyphInstance {
     pub index: GlyphIndex,
     pub point: LayoutPoint,
 }
+
+#[cfg(test)]
+mod test {
+    use super::{FontRenderMode, SubpixelOffset};
+
+    #[test]
+    fn test_subpx_quantize() {
+        let rm = FontRenderMode::Subpixel;
+
+        assert_eq!(rm.subpixel_quantize_offset(0.0), SubpixelOffset::Zero);
+        assert_eq!(rm.subpixel_quantize_offset(-0.0), SubpixelOffset::Zero);
+
+        assert_eq!(rm.subpixel_quantize_offset(0.1), SubpixelOffset::Zero);
```

Comment:
nit: could be nicer if those `subpixel_quantize_offset` calls are batched per expected `SubpixelOffset`